src/python/WMCore/FwkJobReport/Report.py

```diff
@@ -192,7 +192,7 @@ def jsonizePerformance(perfSection):
         Convert the performance section of the FWJR into JSON.
         """
         jsonPerformance = {}
-        for reportSection in ["storage", "memory", "cpu", "multicore"]:
+        for reportSection in ["storage", "memory", "cpu", "multicore", "cmssw"]:
```

Comment:
This very same line, I believe, has been changed also in PR: [#11663](https://github.com/dmwm/WMCore/pull/11663/files#diff-fefc575f5c117d526809c2c64becf519c53dc111d23ffe8b8989881ac3a6052eR195), so I'd expect some collisions here. Whoever is about to be merging this, should pay attention.